io.github.Soundux.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
```

Comment:
The desktop file, icons & metainfo/appdata files should be upstreamed if possible

Reply:
Okay, I'll commit them there and update the tag